tests/ImageSharp.Tests/TestUtilities/TestEnvironment.Features.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Tests
+{
+    public static partial class TestEnvironment
+    {
+        internal static class Features
+        {
+            public const string On = "1";
+            public const string Off = "0";
+
+            public const string EnableAES = "COMPlus_EnableAES";
+            public const string EnableAVX = "COMPlus_EnableAVX";
+            public const string EnableAVX2 = "COMPlus_EnableAVX2";
+            public const string EnableBMI1 = "COMPlus_EnableBMI1";
+            public const string EnableBMI2 = "COMPlus_EnableBMI2";
+            public const string EnableFMA = "COMPlus_EnableFMA";
+            public const string EnableHWIntrinsic = "COMPlus_EnableHWIntrinsic";
+            public const string EnableIncompleteISAClass = "COMPlus_EnableIncompleteISAClass";
```

Comment:
This one isn't a production switch.

For the others, there is a rough hierarchy they follow:
* EnableHWIntrinsic
  * EnableSSE
    * EnableSSE2
      * EnableAES
      * EnablePCLMULQDQ
      * EnableSSE3
        * EnableSSSE3
          * EnableSSE41
            * EnableSSE42
              * EnablePOPCNT
              * EnableAVX
                * EnableFMA
                * EnableAVX2
  * EnableBMI1
  * EnableBMI2
  * EnableLZCNT

`FeatureSIMD` ends up impacting all SIMD support (including `System.Numerics`) but not things like `LZCNT`, `BMI1`, or `BMI2`
`EnableSSE3_4` is a legacy switch that exists for compat and is basically the same as `EnableSSE3`

Reply:
The hierarchy basically means disabling SSE (`EnableSSE=0`) will also disable everything below it

Reply:
Ah lovely. I'll reference that. I intend on doing more intrinsics work in the near future.